.changes/drag-drop-events-renamed-api.md

```diff
@@ -0,0 +1,17 @@
+---
+"@tauri-apps/api": "patch:breaking"
+---
+
+Renamed drag and drop events in `TauriEvent` enum to better convey when they are triggered:
+
+- `TauriEvent.DRAG` -> `TauriEvent.DRAG_ENTER`
+- `TauriEvent.DROP_OVER` -> `TauriEvent.DRAG_OVER`
+- `TauriEvent.DROP` -> `TauriEvent.DRAG_DROP`
+- `TauriEvent.DROP_CANCELLED` -> `TauriEvent::DRAG_LEAVE`
```

Comment:
'drag leave' makes sense on the JS world because you must define which elements receive the event; in our case 'leave' is misleading because it's not triggered when the drag operation moves out of the window

Reply:
It should be triggered, if not, that's a bug

Reply:
oohhh i didn't know we changed that on tao, let me check

Reply:
works indeed, but now i miss the old https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragend_event from https://docs.rs/wry/0.29.0/wry/webview/enum.FileDropEvent.html#variant.Cancelled

Reply:
we can't differentiate between cancelled and leave so we combined them, not sure what's the best name here